.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    browser: true,
```

Comment:
How come this is set? Isn't this purely Node?

Reply:
yea, it got produced when I did eslint init. Removed